### DISCUSSION TITLE
Implement an option to fill selection natively.

### DISCUSSION THREAD

**Discussion by JaylyDev** [Mojang/minecraft-editor#81] [Feedback]
The editor seems to use the scripting API to fill blocks. There is a possibility that performance might slow down (and leads to hang?) when it comes to filling large amount of chunks. Consider implement an option to fill a selection natively for better performance.

**Comment by dacowan** [comment]
We will be working to optimize large block operations from the API, but in the meantime you can still use the `/fill` command
